mods/cnc/maps/gdi10a/map.yaml

```diff
@@ -0,0 +1,898 @@
+MapFormat: 11
+
+RequiresMod: cnc
+
+Title: 10a: Orcastration
+
+Author: Westwood Studios
+
+Tileset: TEMPERAT
+
+MapSize: 64,64
+
+Bounds: 6,5,55,57
+
+Visibility: MissionSelector
+
+Categories: Campaign
+
+LockPreview: True
+
+Players:
+	PlayerReference@Neutral:
+		Name: Neutral
+		OwnsWorld: True
+		NonCombatant: True
+		Faction: Random
+		Allies: Neutral
+		Enemies: Nod, GDI
+	PlayerReference@neutral:
+		Name: neutral
+		Faction: gdi
+		Allies: neutral
+		Enemies: Nod
```

Comment:
Actually, I went back and played the original on a WinXP Virtual Machine.  I was surprised that the civilians are enemies of both GDI and Nod.  Maybe I extracted this from GENERAL.MIX with van der Spek's XCC tool? I don't remember.

Since I can find no civilians on the team, I guess we can remove the team.

Reply:
Oops no, there's a bunch of `neutral` _and_ `Neutral` actors including the sandbags.  I will leave it in the state that it was generated by `OpenRA.Utility`.